fluent/src/parser.js

```diff
@@ -310,8 +312,13 @@ class RuntimeParser {
       eol = this._length;
     }
 
-    const firstLineContent = start !== eol ?
-      this._source.slice(start, eol) : null;
+    // If there's any text between the = and the EOL, store it for now. The next
+    // non-empty line will decide what to do with it.
+    const firstLineContent = start !== eol
+      // Trim the trailing whitespace in case this is a single-line pattern.
+      // Multiline patterns are parsed anew by getComplexPattern.
```

Comment:
It would be nice to one day be able to pass first line to complex string and go from there.

Reply:
It's doable, but keep in mind that we'd need to parse `firstLineContent`  in `getComplexPattern` anyways in case it contains placeables or escape sequences. The current approach of rewinding the index to the beginning of the first line works pretty well IMO.